rust-src/concordium_base/src/protocol_level_tokens/token_operations.rs

```diff
@@ -14,13 +14,9 @@ use concordium_contracts_common::AccountAddress;
 /// Module that implements easy construction of protocol level token operations.
 ///
 /// The operations are composed to transactions via
-/// [`construct::token_holder_operations`](crate::transactions::construct::token_holder_operations)
-/// and
-/// [`construct::token_governance_operations`](crate::transactions::construct::token_governance_operations).
+/// [`construct::token_update_operations`](crate::transactions::construct::token_update_operations)
```

Comment:
Other types updated in the haskell code base already (which needs to be reflected in the rust-part of the code)

- governance account is removed from CreatePLT (e.g. events)
- governance account is added to the initialization parameters
- `governance account` is a `tokenHolder` (not an account address) in the init parameter.
- updated From/ToJSON instances for unified transactions
- reject reasons got updated.
- `TokenState` query does not include the `governance account` anymore.
- `TokenModule` query does now include the `governance account`.